databases/week3/Assignmnet_Task.sql

```diff
@@ -0,0 +1,104 @@
+-- Active: 1660903523654@@127.0.0.1@3306@database_week3
+USE database_week3;
+-- Queries : Meal TABLE
+SELECT * FROM `Meal`;
```

Comment:
Remove parenthesis **SELECT * FROM `Meal`;** around table names. It doesn't run on vs code. it should be like this `SELECT * FROM Meal;` This applies to all codes of the line where you put parenthesis around table names.

Reply:
I will remove backtit `` .  Thank you for feedback